### ISSUE TITLE
A11Y and Markdown links

### ISSUE THREAD

**Issue by daryldyck7** [jeziellago/compose-markdown#60]
Is there a way to have TalkBack work for the links embedded within MarkdownText? Currently, when double clicking - it does not work for embedded links.
